build/Version.props

```diff
@@ -5,15 +5,20 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(PackageVersion)' == '' ">
+
+    <!-- If $(PB_VersionStamp) is empty and $(PB_IsStable) is not true, then a default PackageBuildQuality will be added. -->
     <PackageReleaseVersion Condition="'$(PackageReleaseVersion)' == '' ">2.1.300</PackageReleaseVersion>
-    <PackageBuildQuality>$(BUILD_QUALITY)</PackageBuildQuality>
-    <PackageBuildQuality Condition=" '$(PackageBuildQuality)' == '' ">preview2</PackageBuildQuality>
+    <PackageBuildQuality>$(PB_VersionStamp)</PackageBuildQuality>
```

Comment:
FYI you don't have to accept this parameter as "PB_VersionStamp" if you don't want.  We can feed it into the build.  If desired in VSTS. Example:

/p:BUILD_QUALITY=$(PB_VersionStamp)

Reply:
Trying to reduce the number of properties that can impact this. 

Is this change required in preview1 or is it okay to have this only starting preview2?

Reply:
Preview2, thanks